### PR TITLE
Adding Cake build tool.

### PR DESCRIPTION
**Pull request by Ghasan**

### CODE REVIEW

**Comment by r15ch13** [comment]
Cool!

File: bucket/cake.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "license": "MIT",
+    "url": "https://github.com/cake-build/cake/releases/download/v0.23.0/Cake-bin-net461-v0.23.0.zip",
+    "hash": "1ad334de4188e6dd320a8501dd0d76f5efc8f2f4e83a562fb3f309a298f39a4a",
+    "bin": "cake.exe",
+    "suggest": {
+        "NuGet": [
+            "nuget"
+        ]
+    },
+    "checkver": {
```
Comment (rasa) [inline bucket/cake.json]:
````
”checkver”: {
“github”: “https://github.com/cake-build/cake”
}
````
Would,work, as well

Comment (r15ch13) [reply bucket/cake.json]: yup https://github.com/lukesampson/scoop/commit/3cb03bec49daf7d9fa38068ba3d333b3bfc9c204

Comment (Ghasan) [reply bucket/cake.json]: All the builds currently are marked as pre-release, and I thought it is better an explicit regex, as I read that "github" property ignores pre-releases.

https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate

Comment (rasa) [reply bucket/cake.json]: Good point. @r15ch13?

Comment (r15ch13) [reply bucket/cake.json]: https://github.com/cake-build/cake/releases/latest will fall back to the list of pre-releases and then the first result is used. I see no problem there 😄

Comment (Ghasan) [reply bucket/cake.json]: Interesting. Thanks all for the help 👍

**Pull request merged**